### PR TITLE
Overhaul guide to embed videos

### PR DESCRIPTION
**Pull request by felixfbecker**
- Prefer GIF-like videos over real GIFs
- Explain how to add captions to videos
- Prefer self-hosting videos over YouTube

### CODE REVIEW

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-> Note: This guide presumes macOS.
+> NOTE: This guide presumes macOS.
 
 ## TL;DR
 
 - Favor capturing only the part of the screen that’s required
-- Keep GIFs 15 seconds or less
-- GIFs are typically great for short "check this out" recordings, where videos are better at "how-to" or promotional style content
+- Keep GIF-like videos 15 seconds or less
+- Do not actually use real GIF images, record a GIF-like video and embed it in a way that behaves like a GIF.
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:9]:
```suggestion
- Do not actually use real GIF images, record a GIF-like video and embed it in a way that behaves like a GIF
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+- GIF-like videos are typically great for short "check this out" recordings, where videos are better at "how-to" or promotional style content
 - Keep GIF and images as small as possible with no retina sizes (ratio of image size to intended display size is 1:1)
 - If doing a full-screen recording, 1280x720 (720p) will produce easy to read results at default settings
 - If you’re producing visual content regularly, invest in paid apps (you can expense them too):
-  - Chose [Cleanshot](https://getcleanshot.com/) if just doing screenshots and GIFs.
+  - Chose [Cleanshot](https://getcleanshot.com/) if just doing screenshots and GIF-like videos.
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:14]:
```suggestion
  - Choose [Cleanshot](https://getcleanshot.com/) if just doing screenshots and GIF-like videos.
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 3]
```diff
@@ -53,20 +54,20 @@ Whether you need a dedicated screenshot application or not depends on how often
 
 ## Adding a screen recording (screencast)
 
-A screen recording (screencast) is a great way to see a feature or flow in action, and we want everyone at Sourcegraph to feel capable of creating GIFs and/or videos. This doesn’t mean though that you’re on your own, so if you need assistance of any kind, including something created entirely for you, just reach out to your mates on the marketing team and we’ll help you get it sorted!
+A screen recording (screencast) is a great way to see a feature or flow in action, and we want everyone at Sourcegraph to feel capable of creating GIF-like videos and/or longer videos. This doesn’t mean though that you’re on your own, so if you need assistance of any kind, including something created entirely for you, just reach out to your mates on the marketing team and we’ll help you get it sorted!
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:57]:
```suggestion
A screen recording (screencast) is a great way to see a feature or flow in action, and we want everyone at Sourcegraph to feel capable of creating GIF-like videos and/or longer videos. If you need assistance of any kind, including something created entirely for you, just reach out to your mates on the marketing team and we’ll help you get it sorted!
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 4]
```diff
@@ -53,20 +54,20 @@ Whether you need a dedicated screenshot application or not depends on how often
...
 ### Screen recording apps
 
 The reason to use a paid tool is more settings, export quality controls, and a nicer workflow for producing assets. If you’re planning on doing this more than once, it’s worth investing in a professional tool.
 
-- [Cleanshot](https://getcleanshot.com/)<br/>
-  Export screen recordings to GIF or video and nice features such as hiding desktop icons and notifications during recordings automatically. It does not have any editing or annotation tools for screen recordings though.
-
-- [Quicktime (videos only)](https://support.apple.com/en-au/guide/quicktime-player/qtp97b08e666/mac)<br/>
+- [QuickTime (videos and GIF-like videos)](https://support.apple.com/en-au/guide/quicktime-player/qtp97b08e666/mac)<br/>
   Can capture a window, a selection of, or the entire screen, and in conjunction with the "Split Clip" functionality makes trimming the start and end easy. It’s a great tool that’s free on macOS.
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:64]:
```suggestion
  Can capture a window, a selection of, or the entire screen, and in conjunction with the "Split Clip" functionality makes trimming the start and end easy. It’s a great tool, free on macOS.
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 5]
```diff
@@ -53,20 +54,20 @@ Whether you need a dedicated screenshot application or not depends on how often
...
 
+- [Cleanshot](https://getcleanshot.com/)<br/>
+  Export screen recordings to video and nice features such as hiding desktop icons and notifications during recordings automatically. It does not have any editing or annotation tools for screen recordings though.
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:67]:
```suggestion
  Export screen recordings to video and nice features such as hiding desktop icons and notifications during recordings automatically. It does not have any editing or annotation tools for screen recordings.
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 6]
```diff
@@ -77,9 +78,9 @@ Recording a great screencast is a large topic, but here are some essentials to k
 - Only capture the part of the screen that is necessary. If the output is video, then aim for 16:9 ratio for YouTube
 - If the entire screen needs to be captured, then your resolution should be 1280x720 (720p), as that will make on-screen elements easy to read
 - Decide whether it should be standalone, or if it only makes sense when embedded with content
-- Ideally, GIFs should less than 15-20 seconds in length, otherwise, it’s most likely better as a video
+- Ideally, GIF-like videos should less than 15-20 seconds in length, otherwise, it’s most likely better as a video
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:81]:
```suggestion
- Ideally, GIF-like videos should be less than 15-20 seconds. Otherwise, it’s most likely better as a video.
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 7]
```diff
@@ -92,6 +93,7 @@ How to add a great voice-over is a huge topic that encompasses things such as:
 - Having a pleasant-sounding voice
 - Audio recording quality
 - Mixing voice-overs concerns such as equalization, compression, noise gates, and loudness metering
+- Adding captions to make the voice-over accessible to everyone, including people who are deaf or with hearing loss
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:96]:
```suggestion
- Adding captions to make the voice-over accessible to everyone, including people who are deaf or with hearing loss, people who can't use their speakers or headphones in the moment, or people have too much background noise.
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 8]
```diff
@@ -105,18 +107,84 @@ But to get you started, here are some essential tips:
...
-## Embedding GIFs
+## Embedding videos
+
+The video should be uploaded to the [sourcegraph-assets](https://console.cloud.google.com/storage/browser/sourcegraph-assets/?project=sourcegraph-de&folder=true&organizationId=true) on Google Cloud storage and linked to from the URL `https://sourcegraphstatic.com/path/to/file`.
+Use or create subfolders as appropiate, e.g. the `handbook` or `blog` folder.
+
+Videos are embedded in Markdown and HTML using the `<video>` tag:
+
+```html
+<video controls src="https://sourcegraphstatic.com/my-funny-video.mp4"></video>
```
Comment (quinnkeast) [inline handbook/marketing/adding_screenshots_screen_recording.md:158]: Nit: maybe include the entire tag as a template, with the captions track and all?

Comment (felixfbecker) [reply handbook/marketing/adding_screenshots_screen_recording.md:158]: Hmm, I think there are legit cases where a video may not have a caption track, like when the video simply has no audio voice-over. Since it may not be obvious to everyone that you need to remove the `<track>` in that case as not everyone knows HTML, I thought it would be good to have an example for each case (simple videos, GIF-like videos, videos with captions)

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 9]
```diff
@@ -105,31 +107,104 @@ But to get you started, here are some essential tips:
...
+Otter offers a download in `.SRT` format, which can be easily [converted to WebVTT](https://www.happyscribe.com/subtitle-tools/convert-srt-to-vtt).
+You may need to edit the captions afterwards to correct any mistakes, which you can do in [HappyScribe](https://www.happyscribe.com/subtitle-tools/online-subtitle-editor/free) (click "New subtitles" at the top → "OK" → "Select file") or by editing the WebVTT file in a text editor (it's just a text file).
+
+Zoom Cloud recordings give you an automatically-transcribed WebVTT file when you click "Download" in the top right as one of the files being downloaded, which you can use directly as well.
+
+Once you have your `.vtt` file, upload it to [sourcegraph-assets](https://console.cloud.google.com/storage/browser/sourcegraph-assets/?project=sourcegraph-de&folder=true&organizationId=true) next to the video with the same file name (but with a `.vtt` ending).
+
+Once the file uploaded, click the three dots menu on the right of the `.vtt` file's row, click "Edit metadata", paste `text/vtt; charset=UTF-8` into the "Content-Type" field (replacing any content was in the field before) and click "SAVE".
```
Comment (csbailey5t) [inline handbook/marketing/adding_screenshots_screen_recording.md:130]:
```suggestion
Once the file is uploaded, click the three dots menu on the right of the `.vtt` file's row, click "Edit metadata", paste `text/vtt; charset=UTF-8` into the "Content-Type" field (replacing any content was in the field before) and click "SAVE".
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 10]
```diff
@@ -1,16 +1,17 @@
...
+> NOTE: This guide presumes macOS.
 
 ## TL;DR
 
 - Favor capturing only the part of the screen that’s required
-- Keep GIFs 15 seconds or less
-- GIFs are typically great for short "check this out" recordings, where videos are better at "how-to" or promotional style content
-- Keep GIF and images as small as possible with no retina sizes (ratio of image size to intended display size is 1:1)
+- Keep GIF-like videos 15 seconds or less
+- Do not actually use real GIF images, record a GIF-like video and embed it in a way that behaves like a GIF. Only use GIFs where this is not possible, e.g. on social media or in Google slides.
```
Comment (csbailey5t) [inline handbook/marketing/adding_screenshots_screen_recording.md:9]:
```suggestion
- Do not actually use real GIF images. Record a GIF-like video and embed it in a way that it behaves like a GIF. Only use GIFs where this is not possible, e.g. on social media or in Google slides.
```

**Review by csbailey5t — APPROVED** [review]
